i8_terminal/types/condition_param_type.py

```diff
@@ -0,0 +1,59 @@
+import json
+from typing import Dict, List, Tuple
+
+from i8_terminal.common.formatting import format_number_v2
+from i8_terminal.common.metrics import get_all_metrics_df
+from i8_terminal.types.auto_complete_choice import AutoCompleteChoice
+
+PERIOD_TYPES: Dict[str, str] = {
+    "fy": "mry",
+    "q": "mrq",
+    "ttm": "ttm",
+    "ytd": "ytd",
+    "d": "",
```

Comment:
With the new changes to the ES index, we now always have period type. d => d, r => r.